layers/amqp/Dockerfile

```diff
@@ -1,5 +1,6 @@
 ARG PHP_VERSION
 FROM bref/build-php-$PHP_VERSION:1.1.4 AS ext
+ARG PHP_VERSION
```

Comment:
You were missing this line. 

Since this is a "multi image build", we technically start a new image after "FROM", that is why we need to redefine the ARG.

Reply:
Thank you a lot ! I didn't knew it :)